ksicht/pdf.py

```diff
@@ -92,18 +94,42 @@ def envelopes(recipients: List[EnvelopeRecipientInfo], our_lines, out_file):
     return out_file
 
 
-def concatenate(in_files, out_file):
+def concatenate(in_files, out_file, as_duplex=False):
     """Merge all input PDF files into a single out file."""
     writer = PdfWriter()
 
     for f in in_files:
-        writer.addpages(PdfReader(f).pages)
+        current_pdf = PdfReader(f)
+        num_pages = len(current_pdf.pages)
 
-    writer.write(out_file)
+        writer.addpages(current_pdf.pages)
+
+        if as_duplex and (num_pages >= 1) and (num_pages % 2 == 1):
+            # add blank A4 page
+            writer.addpage(get_blank_page())
 
+    writer.write(out_file)
     return out_file
 
 
+def get_blank_page():
+    blank_pdf_filename = settings.BLANK_PDF_FILEPATH
+    if not Path(blank_pdf_filename).exists():
+        writer = PdfFileWriter()
+        writer.add_blank_page(width=8.27 * 72, height=11.7 * 72)
```

Comment:
Prosím použijme `PaperSize` ať je to aspoň trochu readable:

https://pypdf.readthedocs.io/en/latest/modules/PaperSize.html#add-blank-page-with-papersize